### ISSUE TITLE
Bug: Enum::toArray(false) return with default null, if default not set

### ISSUE THREAD

**Issue by Mapteg34** [mad-web/laravel-enum#4]
Have a enum:
```
/**
 * @method static BotKeywordType STRING()
 * @method static BotKeywordType EMAIL()
 * @method static BotKeywordType PHONE()
 */
final class KeywordType extends Enum
{
    //const __default = self::STRING;

    const STRING = 'string';
    const EMAIL = 'email';
    const PHONE = 'phone';
}
```

When use
`var_dump(KeywordType::toArray(false));`

result:
```
array (size=4)
  'STRING' => string 'string' (length=6)
  'EMAIL' => string 'email' (length=5)
  'PHONE' => string 'phone' (length=5)
  '__default' => null
```

But if i set default value - all ok, __default key not present...

**Comment by wfeller** [comment]
It can be solved by replacing the toArray() method like so:

```
public static function toArray(bool $include_default = false): array
    {
        $result = parent::toArray();

        if (! $include_default) { //isset($result['__default']) has been removed
            unset($result['__default']);
        }

        return $result;
    }
```

**Comment by slavarazum** [comment]
Fixed in [1.2.0](https://github.com/mad-web/laravel-enum/releases/tag/1.2.0) version

**Comment by Mapteg34** [comment]
Thank you
